legacy/docker-compose.yml

```diff
@@ -27,12 +27,17 @@ services:
       - ./outputs:/data/outputs
     command: java -Xmx32g -cp ./target/scala-2.12/skema_text_reading-assembly-0.1.0-SNAPSHOT.jar org.ml4ai.skema.text_reading.apps.AnnotatePlainTextFiles -o /data/outputs /data/inputs
   data_format:
-    image: lumai/askem-text-reading-utilities:sha-5273bbb
+    image: lumai/askem-text-reading-utilities:sha-c595e24
```

Comment:
Not sure if you're depending on that specific commit.  Otherwise you can use `latest` which always points to the most recent commit that successfully published (see https://hub.docker.com/repository/docker/lumai/askem-text-reading-utilities/general).